test/e2e/autoscale_test.go

```diff
@@ -132,6 +133,17 @@ func TestTargetBurstCapacity(t *testing.T) {
 		}))
 	test.EnsureTearDown(t, ctx.Clients(), ctx.Names())
 
+	cm, err := ctx.clients.KubeClient.CoreV1().ConfigMaps(system.Namespace()).
+		Get(context.Background(), netpkg.ConfigName, metav1.GetOptions{})
+	if err != nil {
+		t.Fatal("Fail to get ConfigMap config-network:", err)
+	}
+	activatorCA := cm.Data[netpkg.ActivatorCAKey]
```

Comment:
This code confused me, because it made me think that the CA cert was stored in `ActivatorCAKey`.

I'd love to see us rename this to `ActivatorCASecret` if it needs to be a secret instead. In this case, you can probably also do:

```go
if cm.Data[netpkg.ActivatorCAKey] != "" {
   ...
}
```

Rather than extracting to a temporary and then checking the length.